skunkworks_crow/src/main/res/drawable/selector_bottom_recive.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_pressed="true">
+        <shape>
+            <solid android:color="@color/colorPrimary" />
+            <stroke android:width="1dp" android:color="@color/colorPrimary" />
+            <corners android:radius="20dp" />
+        </shape>
+    </item>
+
+    <item>
+        <shape>
+            <solid android:color="	#D3D3D3" />
```

Comment:
Please fix this spacing and add the color to the `colors.xml`

Reply:
Done